### PR TITLE
Ignore vessels that touch the ground or might fall underground, remove altitude thresholds

### PR DESCRIPTION
**Pull request by eggrobin**
Point 5 of the 10-point plan, tested as part of point 6.

### CODE REVIEW

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 1]
```diff
@@ -380,21 +378,19 @@ public partial class PrincipiaPluginAdapter
     if (!(vessel.situation == Vessel.Situations.SUB_ORBITAL ||
           vessel.situation == Vessel.Situations.ORBITING ||
           vessel.situation == Vessel.Situations.ESCAPING ||
-          (vessel.situation == Vessel.Situations.FLYING &&
-           (vessel.altitude > manageability_altitude_threshold ||
-            vessel.altitude == -vessel.mainBody.Radius)))) {
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Shouldn't we keep the `-Radius` condition?  Do we want to manage the vessel at the centre of the celestial.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 2]
```diff
@@ -415,6 +411,27 @@ public partial class PrincipiaPluginAdapter
...
+  }
+
+  private bool TouchesTheGround(Vessel vessel) {
+    return vessel.parts
+        .Where(part =>
+               part.Modules.OfType<ModuleWheelBase>()
+                   .Where(wheel => wheel.isGrounded)
+                   .Any() ||
+               part.currentCollisions
+                   .Where(collider => collider.gameObject.layer == 15)
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Could we have a named constant for `15`?

Comment (eggrobin) [reply ksp_plugin_adapter/ksp_plugin_adapter.cs]: Added an enum.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 1 above)
Comment (eggrobin) [reply ksp_plugin_adapter/ksp_plugin_adapter.cs]: As discussed, this is fine (🔥🐶☕🔥).

**Pull request merged**